services-directions-models/src/main/java/com/mapbox/api/directions/v5/utils/UnrecognizedPropertiesUtils.java

```diff
@@ -0,0 +1,57 @@
+package com.mapbox.api.directions.v5.utils;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import com.google.gson.JsonElement;
+import com.mapbox.auto.value.gson.SerializableJsonElement;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Provides utility methods to work with unrecognized properties.
+ */
+public final class UnrecognizedPropertiesUtils {
```

Comment:
Should this be public? `DirectionsJsonObject` already exposes these accessors.

Reply:
I want to access it from the `service-directions-refresh-models` module. To avoid code duplicating.

Reply:
I imagine we don't need this if we expose the map getters.